### ISSUE TITLE
Could we build a new release for tag 9.1.0 to npm?

### ISSUE THREAD

**Issue by kamechb** [swimlane/ngx-dnd#243]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Sorry, we will not be able to answer every support request.  Please consider other venues for support requests
```

**Current behavior**
The latest version in NPM is 9.0.0 which was published 2 years ago. And that's a build for view engine. 

**Expected behavior**
Since angular 16 removes the view engine support completely, could we release a package for ivy to support angular 16? 

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **ngx-dnd version:** x.x.x

* **Angular version:** 2.x.x

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by eighties8** [comment]
Is there an update on this request? My app cannot be compiled as of Angular 16 with latest 9.0 release of the library

**Comment by Delagen** [comment]
I simply copied Typescript source to subfolder of project, write alias in tsconfig.json, and seems all ok. I tired to wait new version of library and may be replace it in future

**Comment by eighties8** [comment]
@Delagen can you summarize your workaround in a way that it can be reproduced?

**Comment by Delagen** [comment]
> @Delagen can you summarize your workaround in a way that it can be reproduced?
> 
> I was about to try the fork created by user [henrikmiszoglad](https://github.com/henrikmiszoglad):
> https://github.com/henrikmiszoglad/ngx-dnd-ivy

I don't want to try simply fork, project is small, and I simply use as source in my project with simple tsconfig paths alias. I prefer to wait till it will be maintained project.

**Comment by eighties8** [comment]
> I don't want to try simply fork, project is small, and I simply use as source in my project with simple tsconfig paths alias. I prefer to wait till it will be maintained project.

How are you using ngx-dnd library in your project? I'm using it as a dependency of my project which itself is an angular libary. I've copied the src into my library, but I'm having lots of compiler errors when trying to build. I've added the paths to my compilerOptions.paths and references to "@swimlane/ngx-dnd" within my libraries now do appear to be referencing the local copy and no longer reference node_modules, however, the reference to @swimlane/dragula is throwing error. Do you see same?

```
This module can only be referenced with ECMAScript imports/exports by turning on the 'allowSyntheticDefaultImports' flag and referencing its default export.ts(2497)
```
<img width="1445" alt="Screenshot 2024-02-08 at 3 04 22 PM" src="https://github.com/swimlane/ngx-dnd/assets/9481404/6a328dc8-162f-46cd-b171-59ce8514772f">

**Comment by Delagen** [comment]
> > I don't want to try simply fork, project is small, and I simply use as source in my project with simple tsconfig paths alias. I prefer to wait till it will be maintained project.
> 
> @Delagen, really appreciate your help. I'm still stuck on this.
> 
> How are you using ngx-dnd library in your project? I'm using it as a dependency of my project which itself is an angular libary. I've copied the src into my library, but I'm having lots of compiler errors when trying to build. 
> 
> I've added the paths to my compilerOptions.paths and references to "@swimlane/ngx-dnd" within my libraries do appear to be referencing the local copy and no longer reference node_modules, however, the reference to @swimlane/dragula is throwing error. Do you see same?
> 
> ```
> This module can only be referenced with ECMAScript imports/exports by turning on the 'allowSyntheticDefaultImports' flag and referencing its default export.ts(2497)
> ```
> <img width="1445" alt="Screenshot 2024-02-08 at 3 04 22 PM" src="https://github.com/swimlane/ngx-dnd/assets/9481404/6a328dc8-162f-46cd-b171-59ce8514772f">
> 
> 
> 
> 

enable this flag in compilerOptions
